controllers/membersController.js

```diff
@@ -21,6 +22,34 @@ const getMembers = async (req, res) => {
   }
 }
 
+/**
+ * Fetches the data about our inactive/idle members
```

Comment:
`Returns the usernames of inactive/idle members` seems better to me personally, what do you think?